pom.xml

```diff
@@ -1088,6 +1087,10 @@ under the License.
 				<scala.version>2.12.15</scala.version>
 			</properties>
 
+			<modules>
+				<module>flink-tests-java17</module>
+			</modules>
```

Comment:
Shouldn't this belong under java17-target instead? The new module won't compile with other target java versions

Reply:
:+1: true - I always mix them up :facepalm: